source/speech.py

```diff
@@ -436,8 +441,16 @@ def getIndentationSpeech(indentation):
 			res.append(symbol)
 		else:
 			res.append(u"{count} {symbol}".format(count=count, symbol=symbol))
-
-	return " ".join(res)
+		quarterTones += (count*4 if raw[0]== "\t" else count)
+
+		speak = indentConfig & config.INDENT_SPEECH
+		if indentConfig & config.INDENT_TONES: #Remove me during speech refactor.

```

Comment:
We won't be removing this bit during speech refactor. The only thing that will happen is tones.beep will be changed to return a `SpeechCommand`. So, perhaps move the comment down to the tones.beep.